### ISSUE TITLE
The results can not be reproduced

### ISSUE THREAD

**Issue by Vincent-Hoo** [chxy95/Deep-Mutual-Learning#1]
How is the hyper-parameters like?

**Comment by Vincent-Hoo** [comment]
in the KLDivLoss, batchmean is not allowed in pytorch4.0

**Comment by Vincent-Hoo** [comment]
in the KLDivLoss, "batchmean" is not allowed in pytorch0.4

**Comment by chxy95** [comment]
I didn't set the random seed, so the performance may be changeable.
It is the truth that in a training process, especially at the middle and later stages of the process, 
the performance of DML is always better than Indenpent model. 
In the code there is no special parameter need to be adjusted.
You can observe the curve during training if you have puzzles yet.

**Comment by chxy95** [comment]
> How is the hyper-parameters like?

I didn't set the random seed, so the performance may be changeable.
It is the truth that in a training process, especially at the middle and later stages of the process, 
the performance of DML is always better than Indenpent model. 
In the code there is no special parameter need to be adjusted.
You can observe the curve during training if you have puzzles yet.

**Comment by chxy95** [comment]
> How is the hyper-parameters like?

I didn't set the random seed, so the performance may be changeable.
It is the truth that in a training process, especially at the middle and later stages of the process,
the performance of DML is always better than Indenpent model.
There is no special parameter need to be adjusted in the code.
You can observe the curve during training if you still have puzzles.
